.travis.yml

```diff
@@ -5,10 +5,14 @@ node_js:
   - '6'
   - '5'
   - '4'
+  - '0.12'
   - '0.10'
 os:
   - linux
   - osx
 script:
   - npm run ci-test
 sudo: false
+matrix:
+  allow_failures:
+    - node_js: '0.10'
```

Comment:
0.10 is actually reliable enough on Travis. There are random failures on OS X all the time, but Travis makes it easy enough to restart the specific items until they pass. 

I don't think we need this one.

Reply:
Alright, I'll remove